cg/cli/post_process/post_process.py

```diff
@@ -35,4 +35,19 @@ def post_process_sequencing_run(context: CGConfig, run_name: str, dry_run: bool)
     post_processing_service.post_process(run_name=run_name, dry_run=dry_run)
 
 
-post_process_group.add_command(post_process_sequencing_run)
+@post_process_group.command(name="all")
+@DRY_RUN
+@click.pass_obj
+def post_process_all_runs(context: CGConfig, dry_run: bool) -> None:
+    """Post-process all runs from the instruments."""
+    services: PostProcessingServices = context.post_processing_services
+    are_all_services_successful: bool = True
+    for service in [services.pacbio]:
```

Comment:
I guess what we would aim for in the end is to just go "for service in services" here? It is just not possible currently

Reply:
I don't understand, what do you suggest here?